app/src/main/java/com/example/hungrywolfs/fragments/HomeFragment.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.hungrywolfs.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.activityViewModels
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.RecyclerView
+import com.example.hungrywolfs.R
+import com.example.hungrywolfs.adapters.CategoryAdapter
+import com.example.hungrywolfs.adapters.MealAdapter
+import com.example.hungrywolfs.databinding.FragmentHomeBinding
+import com.example.hungrywolfs.model.AppViewModel
+
+class HomeFragment : Fragment() {
+
+    private val viewModel: AppViewModel by activityViewModels()
```

Comment:
activityViewModels() is usually used to get the viewModel instance from the parent activity in order to communicate with other fragments. 
https://developer.android.com/topic/libraries/architecture/viewmodel#sharing

Here you could use **by viewModels()**.